sympy/sets/ordinals.py

```diff
@@ -1,18 +1,16 @@
-from sympy.core import Basic
+from sympy.core import Basic, Integer
 import operator
 
-class OrdinalPow(Basic):
+class OmegaPower(Basic):
     """
     Represents ordinal exponential and multiplication terms one of the
     building blocks of the Ordinal class.
-    In OrdinalPow(a, b) a represents exponent and b represents multiplicity.
+    In OmegaPower(a, b) a represents exponent and b represents multiplicity.
     """
-    def __new__(cls, *terms):
-        if not all (isinstance(i, (int, Ordinal)) for i in terms):
+    def __new__(cls, a, b):
+        if not all (isinstance(i, (int, Ordinal, Integer)) for i in (a, b)):
```

Comment:
It should probably be checked that `b` is of integer type (`int` or `Integer`) and that `a` is converted to an ordinal if it is of integer type.

Reply:
I tried to convert `a` to ordinal, but that results in something like this
```
>>> Ordinal([OmegaPower(1, 1)])
Ordinal(OmegaPower(Ordinal(OmegaPower(Ordinal(()), 1)), 1))
```
Is this fine ?
how to represent them in the tests?

Reply:
cant we leave `a` as integers itself ?